script.js

```diff
@@ -310,3 +310,90 @@ formDsk.addEventListener('submit', (event) => {
     formDsk.submit();
   }
 });
+
+//
+// Use of local storage
+//
+const contact = {
+  name: '',
+  email: '',
+  msg: '',
+};
+
+function populateContactForm() {
+  const currentContact = JSON.parse(localStorage.getItem('data'));
+  titleMob.value = currentContact.name;
+  titleDsk.value = currentContact.name;
+  emailMob.value = currentContact.email;
+  emailDsk.value = currentContact.email;
+  msgMob.value = currentContact.msg;
+  msgDsk.value = currentContact.msg;
+}
+
+function populateStorage() {
+  contact.name = titleMob.value;
+  contact.email = emailMob.value;
+  contact.msg = msgMob.value;
+  const storeData = JSON.stringify(contact);
+  localStorage.setItem('data', storeData);
+
+  populateContactForm();
+}
+
+if (!localStorage.getItem('data')) {
+  populateStorage();
+} else {
+  populateContactForm();
+}
+
+titleMob.addEventListener('input', () => {
+  // update the same field in desktop form
```

Comment:
[OPTIONAL]
In order to respect the JavaScript best practices, and in order for your code not to look cluttered and messy, I kindly ask you to remove all commented code that you have.

Reply:
Thanks for the tip.